internal/api/graphql/graph/baseResolver/component_version.go

```diff
@@ -81,17 +85,23 @@ func ComponentVersionBaseResolver(app app.Heureka, ctx context.Context, filter *
 		case model.ComponentNodeName:
 			componentId = []*int64{pid}
 		}
-	}
-
-	if filter == nil {
-		filter = &model.ComponentVersionFilter{}
+	} else {
+		componentId = lo.Map(filter.ComponentID, func(id *string, _ int) *int64 {
```

Comment:
Can we use this helper?
https://github.com/cloudoperators/heureka/blob/main/internal/util/slice.go#L41-L55